src/main/resources/com/mathworks/ci/tools/MatlabInstaller/help-products.html

```diff
@@ -0,0 +1,15 @@
+<div>
+  <p>
+    (Optional) Products to set up in addition to MATLAB, specified as a list of product names separated by spaces.
```

Comment:
Insert a space-separated list of products to install. The plugin installs the specified products in addition to MATLAB.

For a list of supported products, open the input file for your preferred release from the [`mpm-input-files`](https://github.com/mathworks-ref-arch/matlab-dockerfile/tree/main/mpm-input-files) folder on GitHub. Specify products using the format shown in the input file, excluding the `#product.` prefix. For example, to install Deep Learning Toolbox™ in addition to MATLAB, insert `Deep_Learning_Toolbox` in the **Products** box.

**Example:** Simulink
**Example:** Simulink Deep_Learning_Toolbox

Reply:
(Adding a link to a page that we don't maintain could be risky. An alternative would be to link to the right section of the Plugin Configuration Guide instead.)

Reply:
I agree that links in the help would be bit tricky, however its the accessible data for any plugin user. I feel direct links from help section would help our users aog with having a link in configuration page as well.